### PR TITLE
🎹 Pagination: Create Overview demo page. (jQuery)

### PR DESCRIPTION
**Pull request by siarhei-arzamasau**

### CODE REVIEW

File: apps/demos/Demos/Pagination/Overview/jQuery/index.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    const imageWrapper = createEmployeeImg(employee);
+    const employeeInfo = createEmployeeInfo(employee);
+
+    cardEl.append(imageWrapper);
+    cardEl.append(employeeInfo);
+
+    return cardEl;
+  };
+
+  const renderEmployeeGallery = () => {
```
Comment (Raushen) [inline apps/demos/Demos/Pagination/Overview/jQuery/index.js]: I think we need pageSize and pageIndex arguments here so we could avoid a relation with pagination component.

File: apps/demos/Demos/Pagination/Overview/jQuery/index.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+        if (evt.name === 'pageIndex') {
+          pagination.option('pageIndex', evt.value);
+          renderEmployeeGallery();
+        }
+      },
+    })
+    .dxPagination('instance');
+
+  const createEmployeeImg = (employee) => {
```
Comment (Raushen) [inline apps/demos/Demos/Pagination/Overview/jQuery/index.js]: I'd extract methods to a separate file like employeeGallery.js

**Review by pomahtri — DISMISSED** [review]

File: apps/demos/Demos/Pagination/Overview/jQuery/index.js (hunk 1 above)
Comment (siarhei-arzamasau) [reply apps/demos/Demos/Pagination/Overview/jQuery/index.js]: fixed

File: apps/demos/Demos/Pagination/Overview/jQuery/index.js (hunk 2 above)
Comment (siarhei-arzamasau) [reply apps/demos/Demos/Pagination/Overview/jQuery/index.js]: fixed

**Review by Raushen — DISMISSED** [review]

**Review by Raushen — APPROVED** [review]

**Pull request merged**